scripts/vet.sh

```diff
@@ -40,34 +40,34 @@ not grep 'func Test[^(]' *_test.go
 not grep 'func Test[^(]' test/*.go
 
 # - Check for typos in test function names
-git grep 'func (s) ' -- "*_test.go" | not grep -v 'func (s) Test'
-git grep 'func [A-Z]' -- "*_test.go" | not grep -v 'func Test\|Benchmark\|Example'
+git grep --quiet 'func (s) ' -- "*_test.go" | not grep -v 'func (s) Test'
+git grep --quiet 'func [A-Z]' -- "*_test.go" | not grep -v 'func Test\|Benchmark\|Example'
 
 # - Do not import x/net/context.
-not git grep -l 'x/net/context' -- "*.go"
+not git grep --quiet -l 'x/net/context' -- "*.go"
```

Comment:
Let's just remove this?  It's a symlink now, and there's no reason we would be using it anymore, anyway.

Reply:
Sure. I'm not so sure about it. Removing it from vet.

Reply:
Did I say "symlink"?  I meant "type alias" O_O